src/client/containers/FavoritesPage/FavoritesPage.Container.js

```diff
@@ -27,6 +29,8 @@ const FavoritesPageContainer = () => {
             return (
```

Comment:
Simon, please change `<h1 className="h1-favorites">Favorites page</h1>`
to
`<h1 className="h1-favorites">MY FAVORITES</h1>`
It was my mistake.

Reply:
There is a problem with css
![image](https://user-images.githubusercontent.com/81156179/145110674-7b9dc694-0909-4244-bb88-788a4216664a.png)

But it is not connected with 247 or 231issues. I think we need to create a new one.

Reply:
And please fix my error in App.js file
![image](https://user-images.githubusercontent.com/81156179/145111127-37a77f7f-b250-4e1f-8ba1-b1d2f8481dbd.png)

line 71-74 and 90-93, delete one of them.
Thank you so much!

Reply:
done